src/app/layout.tsx

```diff
@@ -69,6 +70,7 @@ export default async function RootLayout({
           <SessionProvider session={session}>{children}</SessionProvider>
         </PublicEnvProvider>
       </body>
+      <GoogleAnalytics gaId={CONST_GA4_MEASUREMENT_ID} />
```

Comment:
We can still prevent loading the script in the first place when DoNotTrack is enabled using `next/headers`:

```suggestion
      {headers().get("DNT") !== "1" && (<GoogleAnalytics gaId={CONST_GA4_MEASUREMENT_ID} />)}
```

(And maybe we should also check that the `Sec-GPC` header is not `"1"`.)

Reply:
Oh good point, I forgot this is set as a header and not just available via `navigator` in JS. Thanks!

Reply:
I think we have a ticket for GPC, this is very last-minute so I'd like to keep it simple. I'll file one if not